Google.Api.Generator.Tests/ProtoTests/ResourceNames/ResourceNamesResourceNames.cs

```diff
@@ -85,6 +85,97 @@ public static bool TryParse(string simpleresource, out SimpleResourceName result
         public static bool operator !=(SimpleResourceName a, SimpleResourceName b) => !(a == b);
     }
 
+    /// <summary>Resource name for the <c>SimpleInlineResource</c> resource</summary>
+    public sealed partial class SimpleInlineResourceName : gax::IResourceName, sys::IEquatable<SimpleInlineResourceName>
+    {
+        private static readonly gax::PathTemplate s_template = new gax::PathTemplate("items1/{item1_id}/items2/{item2_id}");
+
+        /// <summary>
+        /// Parses the given <c>SimpleInlineResource</c> resource name in string form into a new
+        /// <see cref="SimpleInlineResourceName"/> instance.
+        /// </summary>
+        /// <param name="simpleinlineresource">
+        /// The <c>SimpleInlineResource</c> resource name in string form. Must not be <c>null</c>.
+        /// </param>
+        /// <returns>The parsed <see cref="SimpleInlineResourceName"/> if successful.</returns>
+        public static SimpleInlineResourceName Parse(string simpleinlineresource)
```

Comment:
The parameter name doesn't look right here. Where does this come from?

Reply:
(It's nasty in the other one as well - sorry for missing it before.)

Reply:
Yes, you're right, these need improving.
I'd rather fix this in the next PR if that's OK?

Reply:
Fine by me :)

Reply:
Excellent :)